processing/WordVectors.py

```diff
@@ -0,0 +1,126 @@
+import json
+import random
+import re
+import sys
+
+import matplotlib.pyplot as plt
+import numpy
+import spacy
+from gensim.parsing.preprocessing import preprocess_string, remove_stopwords
+from sklearn.decomposition import PCA
+from sklearn.preprocessing import StandardScaler
+
+
+def get_words(text):
+    """
+    Source: <@FIXME:/somewhere/>
```

Comment:
fix it (add reference to Jackie(/prashant/eugenia/lam/josh/aryn)'s code or import from it)